library/src/main/java/com/venmo/android/pin/PinFragment.java

```diff
@@ -1,130 +1,26 @@
 package com.venmo.android.pin;
 
-import android.app.Activity;
-import android.app.Fragment;
-import android.os.Bundle;
-import android.view.LayoutInflater;
-import android.view.View;
-import android.view.ViewGroup;
+import android.content.Context;
 
+/**
+ * Created by rmercille on 6/13/16.
+ */
+public interface PinFragment {
```

Comment:
Rename this something like  `PinFragmentImplement`? 
PinFragment implies this is a fragment class when this is a interface.